### ISSUE TITLE
Unable to run metamod

### ISSUE THREAD

**Issue by jenrik** [ShootingRange/csgo-container#2]
Servers fails to load plugin metamod

**Comment by jenrik** [comment]
Solution was to install a 32 bit version of libstdc++6 using the command `apt install libstdc++6:i386`.

It was discovered that metamod was complaining about the library being missing in the log file `addons/metamod/bin/metamod-fatal.log`:
```
L 01/25/2021 - 21:01:31: Detected engine 11 but could not load: libstdc++.so.6: cannot open shared object file: No such file or directory
```

**Comment by jenrik** [comment]
Fixed in b83854c98c114f22d88d00a05cf65e74fa01c964
